### PR TITLE
Documentation - Runtime Manager, RAMLs and Cloudhub API ex…

### PR DESCRIPTION
**Pull request by shambhuprasad**
…change

### CODE REVIEW

File: modules/ROOT/pages/lb-create-arm.adoc [hunk 1]
```diff
@@ -36,6 +36,8 @@ Each dedicated load balancer exposes an external domain `_lb-name_.lb.anypointdn
 
 . Select a target Anypoint VPC from the drop-down list.
 
+. Sets the Mule Application Response Timeout value. The default value is 300.
```
Comment (hannanelson) [inline modules/ROOT/pages/lb-create-arm.adoc]: I do not see this in the Create Load Balancer UI.

**Review by nishantkakar — APPROVED** [review]

File: modules/ROOT/pages/lb-create-arm.adoc (hunk 1 above)
Comment (shambhuprasad) [reply modules/ROOT/pages/lb-create-arm.adoc]: Yes. The UI is not released yet in production. However, you can check the UI in our stgx environment. We need to merge this PR after UI goes in production.

Comment (hannanelson) [reply modules/ROOT/pages/lb-create-arm.adoc]: @shambhuprasad - What is the schedule for the change? This kind of UI change will likely affect screenshots - I need to schedule this work. Thanks.

Comment (hannanelson) [reply modules/ROOT/pages/lb-create-arm.adoc]: @shambhuprasad - can we change the label from "Timeout in Seconds" (which doesn't say what the timeout is for) to "Application Timeout" or "Mule Application Timeout"? 

What exactly is this option for? Is it the time to wait for a Mule app before....doing what?

Comment (shambhuprasad) [reply modules/ROOT/pages/lb-create-arm.adoc]: UI change is deployed in production as hotfix as it was escalated. Please review "Load Balancer create/update" to see the updated changes.

**Review by vinayakjangam — APPROVED** [review]

**Review by sarathecoubian — APPROVED** [review]

**Pull request merged**